app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb

```diff
@@ -408,6 +470,34 @@ def self.host_inv_to_lan_hashes(inv, switch_uids)
           result_uids[uid] = new_result
           switch[:lans] << new_result
         end
+
+        dvportgroup_inv.to_miq_a.each do |data|
+          spec = data['config']
+          next if spec.nil?
+
+          # Find the switch to which this lan is connected
+          switch = switch_uids[spec['distributedVirtualSwitch']]
```

Comment:
Yes that was https://github.com/ManageIQ/manageiq/commit/f44b9f093da0c955bfa9719f8ca9f926e7566027